sonar-plugin/org.wso2.code.quality.sonar.plugin/src/main/resources/org/wso2/rules/descriptions/ClassLevelVariablesInServicesCheck.html

```diff
@@ -0,0 +1,21 @@
+<p>Do not declare class level variables in authenticators. Use static and final.</p>
```

Comment:
Let's add the license header

Reply:
Added. Updated existing license headers from 2022 to 2023 as well.